src/appengine/handlers/configuration.py

```diff
@@ -29,7 +29,11 @@
     },
     {
         'name': 'job',
-        'value': data_types.PermissionEntityKind.JOB
+        'value': data_types.PermissionEntityKind.JOB,
```

Comment:
Please git rebase and git pull so that these don't show up.

Reply:
+1, looks like you did `merge` instead of `rebase`, but it's better to avoid `merge`. I can help you to rebase, if needed